Wrapping/Generators/Python/itkExtras.py

```diff
@@ -371,6 +371,17 @@ def GetVnlMatrixFromArray(arr):
 
 vnl_matrix_from_array = GetVnlMatrixFromArray
 
+def GetArrayFromITKMatrix(itk_matrix):
+    return GetArrayFromVnlMatrix(itk_matrix.GetVnlMatrix().as_matrix())
+
+def GetITKMatrixFromArray(arr):
```

Comment:
Awesome !! Thanks, @fbudin69500 !

`GetMatrixFromArray` would be shorter, and the implicit `ITK` I think makes sense since this is an ITK API and it is called with `itk.`.

Also

```
matrix_from_array = GetMatrixFromArray
array_from_matrix = GetArrayFromMatrix
```
Python aliases would we nice.

Reply:
Done.